graylog2-web-interface/src/components/content-packs/ContentPacksList.jsx

```diff
@@ -99,15 +104,17 @@ class ContentPacksList extends React.Component {
         contentPackId={item.id}
         revision={item.rev}
       />);
-      const states = item.states || [];
+
+      const installationCount = this.props.contentPackInstallCount[item.id];
```

Comment:
This doesn't seem to work. I have an installed content pack but the overview doesn't show the badge.

You are trying to get the value for the `item.id` from the `this.props.contentPackInstallCount` object, but the `propTypes` defines this as an object with `contentPackId` and `installCount` fields. Also, the `ContentPacksList` component doesn't seem to receive the `contentPackInstallCount` props because it's always empty for me.

Reply:
Thanks for reviewing @bernd. You are right about the `PropTypes` shape, I used it wrongly. As you can see in the test the format should be `{ 'd3c72e5e-ce15-11e8-a8d5-f2801f1b9fd1': 3 }` (so uid: count).

But the rest seems to be a problem of the server. Did you recompile it?

Reply:
You are right, of course. I didn't restart the server... :see_no_evil: 

The badge looks a bit misplaced though.

![image](https://user-images.githubusercontent.com/461/46871349-6c454b00-ce31-11e8-8db4-47835b07fe86.png)

How about putting it in front of the content pack title? Or closer to the buttons on the right?
That way we wouldn't increase the row size of the content pack entry.

Reply:
I agree. I will play arround and try to find a better solution!